src/components/pokedex.html

```diff
@@ -56,6 +56,17 @@ <h4 style="margin-bottom: 0">Pokédex</h4>
                                 <option value="caught-shiny">Caught Shiny</option>
                             </select>
                         </div>
+
+                        <div class="form-group col-md-6 col-6">
+                            <label for="pokedex-filter-pokerus-status">Pokerus Status</label>
+                            <select id="pokedex-filter-pokerus-status" autocomplete="off" class="custom-select" onchange="PokedexHelper.updateList()">
+                                <option value="all" selected>All</option>
+                                <option value="none">Uninfected</option>
+                                <option value="infected">Infected</option>
+                                <option value="contagious">Contagious</option>
+                                <option value="cured">Resistant</option>
```

Comment:
IMO we shouldn't do it this way, It just makes everything more confusing having it named differently in different places.
This also makes it harder if we change the names in the future as we have to remember this specific area exist.
If you want the names to be different, it should be updated in the enum so we can have it adjusted everywhere.

Reply:
I update the enum for Pokerus and made a change where `Pokerus.None` & `Pokerus.Cured` are use to match with the newest reference.